test/hotspot/jtreg/serviceability/attach/AttachSetGetFlag.java

```diff
@@ -57,6 +57,10 @@ public static void main(String... args) throws Exception {
     testGetFlag("MaxHeapFreeRatio", "60");
     testSetFlag("MaxHeapFreeRatio", "50", "60");
 
+    // Test a non-manageable size_t flag.
+    // Since it is not manageable, we can't test the setFlag functionality.
+    testGetFlag("StringDeduplicationCleanupDeadMinimum", "128");
```

Comment:
A non-experimental flag, like MetaspaceSize, might be better long term in case the experimental flag get removed again.

Reply:
When `MetaspaceSize` is set to 0, following assertion raised at metaspace.cpp: 316
```C++
size_t MetaspaceGC::capacity_until_GC() {
  size_t value = Atomic::load_acquire(&_capacity_until_GC);
  assert(value >= MetaspaceSize, "Not initialized properly?");  // <-----
  return value;
}
```
`LargePageSizeInBytes` is used instead.